shanoir-ng-front/src/app/shared/mass-download/download-setup-alt/download-setup-alt.component.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Shanoir NG - Import, manage and share neuroimaging data
+ * Copyright (C) 2009-2019 Inria - https://www.inria.fr/
+ * Contact us on https://project.inria.fr/shanoir/
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+import { Component, ElementRef, EventEmitter, HostListener, Input, OnInit, Output, ViewChild } from '@angular/core';
+import { UntypedFormBuilder, UntypedFormGroup, Validators } from '@angular/forms';
+import { Format } from 'src/app/datasets/shared/dataset.service';
+import { GlobalService } from '../../services/global.service';
+import { Option } from '../../select/select.component';
+
+@Component({
+    selector: 'download-setup-alt',
+    templateUrl: 'download-setup-alt.component.html',
+    styleUrls: ['download-setup-alt.component.css']
+})
+
+export class DownloadSetupAltComponent implements OnInit {
+
+    @Output() go: EventEmitter<Format> = new EventEmitter();
+    @Output() close: EventEmitter<void> = new EventEmitter();
+    form: UntypedFormGroup;
+    @ViewChild('window') window: ElementRef;
+    @Input() format: Format; 
+    formatOptions: Option<Format>[] = [
```

Comment:
Do not propose BIDS and EEG here.
If we selected BIDS/EEG datasets, they will be downloaded whatever.

Reply:
ok

Reply:
same for the "non alt" download ?